src/components/ViewArticle.jsx

```diff
@@ -1,38 +1,75 @@
 import { useParams } from "react-router-dom";
 import { useState, useEffect } from "react";
-import { getArticle } from "../api";
-import { Link } from "react-router-dom";
+import { getArticle, patchDownvote } from "../api";
+import Comments from "./Comments";
 
 function ViewArticle() {
+  const { id } = useParams();
   const [article, setArticle] = useState([]);
+  const [voteCount, setVoteCount] = useState(0);
+  const [isLoading, setIsLoading] = useState(true);
 
-  const { id } = useParams();
-  const path = `/articles/${id}/comments`;
   if (id === undefined) {
```

Comment:
Not sure when the id would be undefined, but a console.log doesn't qualify as proper error handling anyway as it's unlikely to be noticed by the user. If you want to handle a circumstance where the article doesn't exist, you'd be better doing some logic in the catch block where you are calling your getArticle function (assuming you've set up functionality in your BE project to throw an error when trying to get an article that doesn't exist/making a bad request). It's important to clearly provide visual feedback on the screen for errors for UX and accessibility reasons. Check out the notes on error handling for a couple examples similar to what I ran through yesterday morning in the Q&A. **Note that there is a ticket for general error handling like this later, so I'd recommend carrying on and coming back to this when you get there 😄** https://notes.northcoders.com/courses/js-front-end/error-handling